test/util_test.rb

```diff
@@ -16,6 +16,15 @@ def test_util_shell_escape
     assert_equal test_class.new.escape_for_shell("vm.test.com physical.test.com"), 'vm.test.com\ physical.test.com'
   end
 
+  def test_util_escape_for_cmdexe
+    assert Proxy::Util.instance_methods.include? :escape_for_cmdexe
+
+    test_class = eval "class ProxyUtilTestHelper; include Proxy::Util; end"
+    assert_equal test_class.new.escape_for_cmdexe("Test text &! continues"), ["Test", "text", "&!", "continues"]
```

Comment:
small nit, assert_equal expects the first argument to be the one that we expect (in your case the one you use as the last), that way it constructs more meaningful output when the test fails. The same applies to the line below.